samples/tables_data.md

```
@@ -0,0 +1,36 @@
+# Data Analysis Report
+
+<!-- Quarterly performance data -->
+
+---
+
+# Q1 Sales Summary
+
+| Region | Revenue | Growth |
+|--------|---------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)